feat: Add docs for Device MQTT (#1284)
edgexfoundry/edgex-docs: docs_src/microservices/device/services/device-mqtt/GettingStarted.md

---
title: Device MQTT - Getting Started
---

# Device MQTT - Getting Started

> ## Overview
> 
> Device MQTT is a device service for connecting a device or sensor feed to EdgeX using an MQTT topic.

Reviewer: ```suggestion
## Overview

Device MQTT is a device service for connecting a device or sensor feed to EdgeX using an MQTT protocol.
```

## Running Services with Device MQTT

The simplest way to run all the required services is to use the [Compose Builder](https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}}/compose-builder) tool from a terminal window.

1. Clone [https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}}](https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}})

2. Change to the **compose-builder** folder

3. Run the services
    ```

>     make run ds-mqtt mqtt-broker no-secty ui

Reviewer: UI is now in the base compose as a standard service

```suggestion
    make run ds-mqtt mqtt-broker no-secty
```

    ```

> This runs, in non-secure mode, all the standard EdgeX services, along with a mqtt message broker, device MQTT, and the EdgeX UI.

Reviewer: ```suggestion
This runs, in non-secure mode, all the standard EdgeX services, along with a mqtt message broke and device MQTT.
```
